src/arkouda_server.chpl

```diff
@@ -124,6 +123,48 @@ proc main() {
         stdout.flush();
     }
 
+    /**
+     * Register our server commands in the CommandMap
+     * There are 4 general types
+     * 1. Standard, required commands which adhere to the standard Message signature
+     * 2. Specialized, required commands which do not adhere to the standard Message signature
+     * 3. "Backbone"/default modules we have deemed should be included in every build
+     * 4. "Optional" modules which are included at compilation time via ServerModules.cfg
+     */
+    proc registerServerCommands() {
+        registerBinaryFunction("tondarray", tondarrayMsg);
+        registerFunction("create", createMsg);
+        registerFunction("delete", deleteMsg);
+        registerFunction("arange", arangeMsg);
+        registerFunction("linspace", linspaceMsg);
+        registerFunction("set", setMsg);
+        registerFunction("info", infoMsg);
+        registerFunction("str", strMsg);
+        registerFunction("repr", reprMsg);
+        registerFunction("getconfig", getconfigMsg);
+        registerFunction("getmemused", getmemusedMsg);
+        registerFunction("getCmdMap", getCommandMapMsg);
+        registerFunction("clear", clearMsg);
+
+        // For a few specialized cmds we're going to add dummy functions,
+        // they will be intercepted in the cmd processing select statement
+        registerFunction("array", akMsgSign);
+        registerFunction("connect", akMsgSign);
+        registerFunction("disconnect", akMsgSign);
+        registerFunction("noop", akMsgSign);
+        registerFunction("ruok", akMsgSign);
+        registerFunction("shutdown", akMsgSign);
+
+        // Now Register our "default" collections included in every build
+        import RandMsg, IndexingMsg, OperatorMsg;
+        RandMsg.registerMe();
+        IndexingMsg.registerMe();
+        OperatorMsg.registerMe();
```

Comment:
Woops, my bad, these are already included in `MsgProcessing.chpl` today, so this was me who introduced this error, but still, I think they should be removed from the `ServerModules.cfg` file.

Reply:
Hmm... these were cut-&-pasted from `MsgProcessing.chpl` (see [here](https://github.com/Bears-R-Us/arkouda/pull/1052/files#diff-f0d2928beafa5fed8aa76d7f890411ccb7c860b0ec7075a0e1e3e216b2367ce7L18) )

It seems they are also listed in `ServerModules.cfg` so we have them in there twice.

I'm happy to remove them from here since it amounts to hard-coding them.

Reply:
I think I'm in favor of keeping them in `ServerModules.cfg` because you get an option to include/exclude them yourself and if you're messing around in that file the assumption is that you know what you are doing. @mhmerrill what's your opinion on this?  Force them to always compile or allow the developer to turn them on/off?

Slightly off-topic:  Longer term, we could add some type of dependency management to the `ServerModules.cfg` file since some messages  depend on others (i.e. `KextremeMsg` uses `ArraySetopsMsg`).  If we go that route, we'd likely upgrade the format of the file to something like `ini` or `yml`.  I mean... the compiler will ultimately handle the dependency checking so it may not be needed.

Reply:
Well, with this code (and the code on `master` today) the user does not have an option to include/exclude them, they only think that they do, since we are registering those functions every time anyway, whether or not they are commented out in `ServerModules.cfg` makes no difference. 

I would be in favor of removing them from being default included and just allowing the user to choose everything, but I know that there is an argument for having a set of backbone functionality that is always built.

Reply:
> Well, with this code (and the code on `master` today) the user does not have an option to include/exclude them, they only think that they do, since we are registering those functions every time anyway; whether or not they are commented out in `ServerModules.cfg` makes no difference. (You can test this by commenting those modules out and then making the server and then trying to call `ak.randint()` or something, it will still work)

Exactly.